packages/expo-cli/src/commands/push-creds.ts

```diff
@@ -46,37 +54,63 @@ export default function (program: Command) {
     .command('push:android:show [project-dir]')
     .description('Print the value currently in use for FCM notifications for this project.')
     .asyncActionProjectDir(async (projectDir: string) => {
-      const {
-        args: { remotePackageName },
-      } = await Exp.getPublishInfoAsync(projectDir);
+      log('Logging in...');
+
       let user = await UserManager.getCurrentUserAsync();
-      let apiClient = ApiV2.clientForUser(user);
+      if (!user) {
+        throw new Error('You must be logged in to proceed.');
+      }
 
-      let result = await apiClient.getAsync(`credentials/push/android/${remotePackageName}`);
+      log('Reading project configuration...');
+      const { exp } = ConfigUtils.getConfig(projectDir, { skipSDKVersionRequirement: true });
+      let { username } = user;
+      if (exp.owner) {
+        username = exp.owner;
+      }
 
-      if (result.status === 'ok' && result.fcmApiKey) {
-        console.log(JSON.stringify(result));
+      const isProxyUser = username !== user.username;
+      log(
+        `Getting API key on Expo's servers for project ${exp.slug}${
+          isProxyUser ? ` on behalf of ${username}` : ''
+        }...`
+      );
+      let apiClient = ApiV2.clientForUser(user);
+      const { fcmApiKey } = await apiClient.getAsync(
+        `credentials/android/push/@${username}/${exp.slug}`
+      );
+      if (fcmApiKey) {
+        console.log(`FCM Api Key: ${fcmApiKey}`);
       } else {
-        throw new Error('Server returned an invalid result!');
+        console.log(`There is no FCM Api Key configured for this project`);
```

Comment:
Please use the `log` module (`import log from '../log'`). And probably should exit with a non-zero status here?